transport_nantes/photo/models.py

```diff
@@ -63,6 +69,81 @@ class CategoryType(models.TextChoices):
         verbose_name="Date de soumission",
         auto_now_add=True,
     )
+    # Photo entries are pending (null) until they are accepted (True)
+    # or rejected (False)
```

Comment:
A useful comment would not explain what a boolean is (;-)) so much as explain the context, that this boolean expresses the outcome of a judging process, for example.

Reply:
I added this comment because we usually expect Bools to be either false or true, and this one can be null. 
Also "accepted" sounds to me like we judged it, but I'll word it anyway, comments are cheap hehe

Reply:
It's usually better to err on the side of too much commentary than too little.  Though even better to hit the right Goldilocks ratio. ;-)